examples/streamz/python/sequence_classifier.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import dask
+import cudf
+from clx_streamz_tools import utils
+from clx_streamz_tools import streamz_workflow
+
+
+class SequenceClassifierWorkflow(streamz_workflow.StreamzWorkflow):
```

Comment:
We can still use the name `PhishingDetectionWorkflow` since phishing detection is the use case. `SequenceClassifier` can apply to multiple use cases.

Reply:
same with filename. we can name it `phishing_detection.py`. thanks!

Reply:
@efajardo-nv reverted file and class name to phishing detection. thank you!